jbw/visualizer/Makefile

```diff
@@ -84,7 +84,7 @@ $(BIN_DIR)/%.spv: %.spv
 	cp $< $@
 
 jbw_visualizer: bin $(LIBS) $(JBW_VISUALIZER_OBJS) $(JBW_VISUALIZER_SHADERS:%=$(BIN_DIR)/%)
-		$(CPP) -o $(BIN_DIR)/jbw_visualizer $(CPPFLAGS) $(LDFLAGS) $(JBW_VISUALIZER_OBJS)
+		$(CPP) -o $(BIN_DIR)/jbw_visualizer $(JBW_VISUALIZER_OBJS) $(CPPFLAGS) $(LDFLAGS)
```

Comment:
Just curious, why did you need to change the order of arguments here?

Reply:
I was running into weird issues with compiling the visualizers when the linked libraries were before referenced before the cpp file itself - something about an undefined reference?

Reply:
Oh right, I think libraries are supposed to be specified after object files. I'll change that elsewhere too.